### DISCUSSION TITLE
Changes are not reflected in url  after deployment

### DISCUSSION THREAD

**Discussion by Karan-S-dev** [community/community#33232] [Apps, API and Webhooks]
I have deployed a react app using github actions in azure.

When a new change is pushed using github api, the deployment gets initiated and it is getting success.

But the changes are not reflecting in the url. It is taking sometimes 30 minutes to reflect and sometimes it is taking hours to deploy.
What is the factor determining this time stamp?

I am not able to figure out this issue.

**Comment by jfullstackdev** [comment]
are you checking that in Incognito mode ? maybe it's cached data that you are seeing

if that is really taking hours to deploy, well, yes, that's a possibility even just like 24 hours,
